Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Build the go application into a binary
 FROM golang:alpine as builder
-RUN apk --update add ca-certificates
+RUN apk --update --no-cache add ca-certificates curl
```

Comment:
Is there really any point in installing curl on the builder image, since the builder's only really used to generate the binary and move it to the next stage (scratch/empty image)?